awx_collection/test/awx/test_inventory.py

```diff
@@ -3,20 +3,26 @@
 
 import pytest
 
-from awx.main.models import Inventory
+from awx.main.models import Inventory, Credential
+from awx.main.tests.functional.conftest import insights_credential, credentialtype_insights
 
 
 @pytest.mark.django_db
-def test_inventory_create(run_module, admin_user, organization):
+def test_inventory_create(run_module, admin_user, organization, insights_credential):
+    # Create an insights credential
+
     result = run_module('tower_inventory', {
         'name': 'foo-inventory',
         'organization': organization.name,
         'variables': {'foo': 'bar', 'another-foo': {'barz': 'bar2'}},
+        'insights_credential': insights_credential.name,
         'state': 'present'
     }, admin_user)
+    assert not result.get('failed', False), result.get('msg', result)
```

Comment:
ha, I know it's a test I wrote early on when it doesn't have the recipe to correctly assert for success